porcupine/plugins/autocomplete.py

```diff
@@ -536,7 +536,9 @@ def on_new_filetab(tab: tabs.FileTab) -> None:
     completer.popup.treeview.bind("<Button-1>", (lambda event: completer._accept()), add=True)
 
     # avoid weird corner cases
-    tab.winfo_toplevel().bind("<FocusOut>", (lambda event: completer._reject()), add=True)
+    # On Windows, <FocusOut> runs before treeview click handler, must accept when clicked
+    tab.winfo_toplevel().bind("<FocusOut>", (lambda event: tab.after_idle(completer._reject)), add=True)
```

Comment:
Toplevels get notified of their children's events. Can we instead just ensure it's the main window that is getting unfocused here?

Reply:
Not possible. All focusing events happen before.